miio/integrations/vacuum/roborock/vacuumcontainers.py

```diff
@@ -656,6 +656,32 @@ def dustbin_auto_empty_used(self) -> Optional[int]:
             return self.data["dust_collection_work_times"]
         return None
 
+    @property
+    @sensor(
+        "Strainer used",
+        icon="mdi:air-filter",
+        entity_category="diagnostic",
+        enabled_default=False,
+    )
+    def strainer_used(self) -> Optional[int]:
+        """Return ``strainer_work_times``"""
```

Comment:
```suggestion
    def strainer_cleaned_count(self) -> Optional[int]:
        """Return strained cleaned count."""
```
It's a bit unclear what does this mean? Is it a count of times the strainer has been cleaned? If yes, maybe change the wording.